it/src/test/java/com/google/cloud/teleport/it/bigtable/BigtableResourceManagerUtilsTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.it.bigtable;
+
+import static com.google.cloud.teleport.it.bigtable.BigtableResourceManagerUtils.generateDefaultClusters;
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import com.google.cloud.bigtable.admin.v2.models.StorageType;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+/** Unit tests for {@link com.google.cloud.teleport.it.bigtable.BigtableResourceManagerUtils}. */
+@RunWith(JUnit4.class)
+public class BigtableResourceManagerUtilsTest {
+  @Rule public final MockitoRule mockito = MockitoJUnit.rule();
+
+  private static final String TEST_ID = "test-id";
+  private static final String ZONE = "us-central1-a";
+  private static final int NUM_NODES = 1;
+  private static final StorageType STORAGE_TYPE = StorageType.SSD;
+
+  @Test
+  public void testGenerateDefaultClustersShouldWorkWhenAllParametersValid() {
+    Iterable<BigtableResourceManagerCluster> cluster =
+        generateDefaultClusters(TEST_ID, ZONE, NUM_NODES, STORAGE_TYPE);
+    assertEquals(cluster.iterator().next().getClusterId(), TEST_ID + "-c1");
```

Comment:
nitpick: Truth is preferable.

Reply:
also, maybe saving the BigtableResourceManagerCluster instance and doing the assertions on it will look simpler, instead of `.iterator().next()` on every line